### PR TITLE
Change sane_repr to be logging_repr and add logging_dict beside it.

### PR DESCRIPTION
**Pull request by JTCunning**
@getsentry/infrastructure

### CODE REVIEW

File: src/sentry/db/models/base.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 from .manager import BaseManager
```
Comment (JTCunning) [inline src/sentry/db/models/base.py]: This is pretty much the only important part.

Everything else is just sed.

File: src/sentry/db/models/base.py [hunk 2]
```diff
@@ -109,7 +126,9 @@ class Model(BaseModel):
     class Meta:
         abstract = True
 
-    __repr__ = sane_repr('id')
+    __sane__ = ('id')
```
Comment (mattrobenolt) [inline src/sentry/db/models/base.py]: This isn't a `tuple`, it's only a `str`.

Comment (JTCunning) [inline src/sentry/db/models/base.py]: This was a sed overflow. Shouldn't have actually changed.

**Comment by JTCunning** [comment]
Incomplete idea.

**Pull request closed without merging**